meetings/2022-12-07.md

```diff
@@ -0,0 +1,165 @@
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+* Jeff Waters
+* Wouter Termont
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+### Previous Meeting
+URL: https://github.com/solid/specification/blob/main/meetings/2022-11-30.md
+
+* SC: Consensus both proposing and how we accept them. We have prior consent, previous minutes are PR'd, once reviewed, we have approval but we link to acknowledge that to review or to raise objection to any part of prior meetings. Some concerns or comments, this is your chance. We acknowledge that. I'll pause if you have something to say. (none)
+
+### Current Meeting
+URL: https://github.com/solid/specification/pull/489
+
+* SC: Proposed agenda.
+* SC: Proposed agenda is here. Quick summary, once we have consent, we can get to the body of the topics, and with that we can adjust the topics or add new ones and so on.  So today, we have lightweight. maybe agenda, one topic is on W3C Calendar, just about receiving notifications about community events, this is weekly but other calls, WebID-Profile, all the panel meetings, notifications is up now. Once we have agreement on other meetings, we'll put them up on the agenda. Most people are getting the notifications.
+* SC: We'll have time to talk about special topics, and we have topic of draft working group charter on Solid.
+
+### W3C Calendar
+URL: https://www.w3.org/groups/cg/solid/calendar
+
+* SC: If you are not receiving notifications or updates, like when I update the event, you should be getting this from W3C, if not, let me know or review your own preferences, the steps are:
+* SC: Email notifications of group events are enabled by default. Check settings:
+  1. [Login](https://www.w3.org/accounts/login)
+  2. [My Calendar](https://www.w3.org/users/myprofile/calendar)
+  3. Preferences
+  4. Check the box "Receive email notifications of events to which my groups are invited".
+* SC: The announcements about the agenda are in the Gitter chat, as well as the PR. I'll pause if anyone has comments.
+* PA: (muted)
+* MB: I was wondering if it was possible to create one event that is recurring rather than so many notifications?
+* SC: When the event is created and recurring, ... you will not get notification for recurring events unless the particular event is updated.
+* MB: Ok. I'm wondering if that is what is happening, I receive too many of them.
+* SC: In order to include agenda in event calendar, I do need to update it.
```

Comment:
I wasn't on the call, so this is not a change suggestion, but a relevant comment.

In most groups I'm in, we've found that including a *link* to the agenda in the calendar is better, because it goes in once, and any changes are just made to that remote document -- which do not trigger calendar notification email.  The agenda may be sent to the relevant email list, and its archive URL used for the link; or the agenda may continue to be pushed to GitHub, and a link to the relevant PR used in the calendar.

Reply:
Either way, a notification is distributed, whether by the W3C calendar system, an email to the mailing list, or some other way.

Alternatives:

* For people using the W3C CG's ICS, an update to the W3C calendar including the link to the agenda (without explicitly sending a notification out about the update) will suffice. They'll see the updated information when their local calendar synchronises with W3C's.

* A persistent URL for the agenda, e.g., `https://solidproject.org/w3c-solid-cg-upcoming-agenda` that redirects to the GitHub PR URL can be fixed in the W3C calendar. Requires a person or a script to change the redirect URL.